x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/Md5UnavailableTests.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.expression.function.scalar.string;
+
+import org.elasticsearch.test.ESTestCase;
+import org.elasticsearch.xpack.esql.VerificationException;
+import org.elasticsearch.xpack.esql.core.expression.Literal;
+import org.elasticsearch.xpack.esql.core.tree.Source;
+
+import java.security.Provider;
+import java.security.Security;
+
+import static org.hamcrest.Matchers.notNullValue;
+
+public class Md5UnavailableTests extends ESTestCase {
```

Comment:
NIT: we usually put tests that do not require parametrization in *StaticTests, in this case it would be `Md5StaticTests`.

Reply:
Thanks, will do that.